src/include/wiredtiger.in

```diff
@@ -822,23 +822,12 @@ struct __wt_cursor {
 #endif
 };
 
-#ifdef HAVE_CONTROL_POINT
+#if defined(HAVE_CONTROL_POINT) || defined(SWIG)
 /*!
```

Comment:
Move new lines 826-829 between new lines 66 and 67. Delete the other lines from new line 825-831.